src/de/grobox/liberario/TransportNetworks.java

```diff
@@ -579,4 +610,12 @@ TransportNetwork getTransportNetwork(String idString) {
 	private String getString(int res) {
 		return context.getString(res);
 	}
+
+	private String region(String name, String flag) {
+		return flag + " " + name;
+	}
+
+	private String description(String desc, String networks) {
```

Comment:
That's a good point, thank you!
I did this in an earlier version, however I changed that. That was after I saw the feature of Android-Studio that it can "autocomplete" the strings. (It shows previews like in the screenshot)
![screen](https://cloud.githubusercontent.com/assets/4189642/19014224/7ebc6e68-87e7-11e6-872f-ce40a995ddda.png)
The benefit of using directly the ints is of course less boilerplate-code.

I am not sure and I see benefits in both. Just tell me what you like more and I will patch it if needed.